### PR TITLE
omnibus: scroller perf, devex enhancements 

### LINKED ISSUE #1200: chat: sending a message doesn't scroll to bottom sometimes
**Describe the bug, optionally with screenshots**
Sending a message with a `?msg=...` always scrolls to the focused msg and not the bottom.

**To Reproduce**
1. Click a reference link
2. Then try to send a message in the same channel
3. Notice window stays scrolled on previous focused message from ref link.

**Expected behavior**
Window should scroll to bottom when sending a message

### PR DESCRIPTION
**Pull request by tomholford**
# Context

This addresses #1187. It also includes additional performance enhancements and  devex tweaks.

# Changes

- [x] consistent localStorage key for prerelease banner
- [x] reuse firstUnread hook
- [x] virtual scroller performance enhancements
  - [x] dynamic page fetching - short initial page for quick render, then larger as user scrolls back
  - [ ] detect high acceleration and pause / prefetch higher amounts
     - initially attempted to implement using virtuoso scroll seek, ran into a known issue (https://github.com/petyosi/react-virtuoso/issues/410)
     - now [trying an approach](https://github.com/petyosi/react-virtuoso/issues/410#issuecomment-1320405297) recommended by the maintainer of `react-virtuoso`
- [x] remove channel prefetching; use `last.brief` to sort by recent instead 
- [x] upgrade `react-virtuoso` from 2.19.0 --> 3.1.4 to fix a [scroller bug](https://github.com/petyosi/react-virtuoso/issues/788) on iOS Chrome
- [ ] refactor scrollTo to fix #1200

### CODE REVIEW

File: ui/src/logic/usePrefetchChannels.ts [hunk 1]
```diff
@@ -39,8 +43,20 @@ export default function usePrefetchChannels(flag: string) {
...
 
   const fetchAll = useCallback(async () => {
-    await asyncForEach(Object.keys(group?.channels ?? {}), fetchChannel);
-  }, [fetchChannel, group]);
+    // first, prioritize the recent channel
+    if (recentChannel && recentChannel !== '') {
+      await fetchChannel(recentChannel);
+    }
+
+    // defer the rest, in batches
```
Comment (patosullivan) [inline ui/src/logic/usePrefetchChannels.ts:51]: Thank you!

**Review by patosullivan — APPROVED** [review]
Looks good and runs well locally.

**Review by arthyn — CHANGES_REQUESTED** [review]
Testing this is giving me some really bad behavior just putting this here now before I can write it all out

**Comment by arthyn** [comment]
So I haven't been able to capture this on camera, but in either watercooler or urbit app dev, on a fresh reload, if you scroll up as soon as you see messages and then scroll up again once the second set of messages load, sometimes you get stuck in like a purgatory of sorts where messages are almost all missing and going either up or down scrolls in the wrong direction.

Other than that, it seems like scrolling backwards now gets stuck more often than before.

**Comment by tomholford** [comment]
@arthyn Okay thanks, will look into this

**Review by arthyn — APPROVED** [review]

**Pull request merged**